src/Makefile

```diff
@@ -2,15 +2,7 @@
 MODULE_TOPDIR = ..
 
 # display gui misc
-SUBDIRS = \
-	db \
-	display \
-	general \
-	imagery \
-	raster \
-	raster3d \
-	vector \
-	temporal
+SUBDIRS := $(shell find ./ -maxdepth 2 -mindepth 2 -name "Makefile" -exec dirname {} \; | xargs -I {} basename {} )
```

Comment:
Find command omit [gui ](https://github.com/OSGeo/grass-addons/tree/grass8/src/gui/wxpython) and [hadoop ](https://github.com/OSGeo/grass-addons/tree/grass8/src/hadoop/hd) subdirectory, which contains addons too. These directories are missing from original list of subdirs.

https://github.com/OSGeo/grass-addons/blob/fcbb27b383d61aab40977060f1d2a6ca72184a94/src/Makefile#L5-L13

Reply:
Looks good to me now.